app/views/projects/destroy_info.html.erb

```diff
@@ -27,18 +27,20 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 <%= toolbar title: l(:label_confirmation) %>
-<div class="warning">
-  <p><strong><%=h @project_to_destroy %></strong><br />
-    <%=l(:text_project_destroy_confirmation)%>
-    <% if @project_to_destroy.descendants.any? %>
-      <br />
-      <%= l(:text_subprojects_destroy_warning, content_tag('strong', h(@project_to_destroy.descendants.collect{|p| p.to_s}.join(', ')))).html_safe %>
-    <% end %>
-  </p>
-  <p>
-    <%= form_tag(project_path(@project_to_destroy), :method => :delete) do %>
-      <label><%= check_box_tag 'confirm', 1 %> <%= l(:general_text_Yes) %></label>
-      <%= submit_tag l(:button_delete), class: 'button -highlight' %>
-    <% end %>
-  </p>
+<div class="notification-box -warning -severe">
+  <div class="notification-box--content">
+    <p><strong><%= l(:text_project_destroy_title) %></strong><br />
+      <%= l(:text_project_destroy_confirmation, identifier: @project_to_destroy.identifier)%>
+      <% if @project_to_destroy.descendants.any? %>
+        <br />
+        <%= l(:text_subprojects_destroy_warning, content_tag('strong', h(@project_to_destroy.descendants.collect{|p| p.to_s}.join(', ')))).html_safe %>
+      <% end %>
+    </p>
+    <p>
+      <%= form_tag(project_path(@project_to_destroy), :method => :delete) do %>
+        <label><%= check_box_tag 'confirm', 1 %> <%= l(:general_text_Yes) %></label>
+        <%= submit_tag l(:button_delete), class: 'button -highlight' %>
```

Comment:
This is where we will always struggle with accessibility vs. themability...

You mean that the button background-color might clash with the background-color of the notification in case anyone (no names ^^) has a red-ish highlight button, don't you?

Not sure how we can solve this problem in general...